roles/kafka_controller/tasks/secrets_protection.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Create Kafka Controller Config with Secrets Protection
```

Comment:
Let's add a scenario to validate secrets protection with the controller as well. (Can be done later as a different PR)